include/SemanticModel.h

```diff
@@ -110,6 +114,10 @@ class SemanticModel {
 
     // cache of simple integral types keyed by {width, signedness, 4-state, isReg}
     std::unordered_map<uint64_t, const TypeSymbol*> integralTypeCache;
+
+    // An empty declaration table to make it easier to create SemanticModels
+    // where we don't care about the full compilation process.
```

Comment:
It seems like slang is in general super inconsistent about using doxygen comments, but the fact that you added two comments in this change, one // and one ///, makes it especially noticeable.